com.eminfedar.vaktisalah-gtk.json

```diff
@@ -0,0 +1,101 @@
+{
+    "id": "com.eminfedar.vaktisalah-gtk",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "44",
+    "sdk": "org.gnome.Sdk",
+    "sdk-extensions" : [
+        "org.freedesktop.Sdk.Extension.rust-stable"
+    ],
+    "command": "vaktisalah-gtk",
+    "finish-args": [
+        "--share=network",
+        "--socket=pulseaudio",
+        "--share=ipc",
+        "--socket=x11",
```

Comment:
if it really supports wayland then this should be `fallback-x11`

Reply:
Still needs removing.